charts/langsmith/values.yaml

```diff
@@ -77,6 +77,7 @@ backend:
 #        cpu: 200m
 #        memory: 500Mi
     command: []
+    extraContainerConfig: {}
```

Comment:
is this a breaking change for helm chart users?

Reply:
nope, none of these are breaking just allow specifying some additional things